tasks/connectors/qualys_was/qualys_was.rb

```diff
@@ -21,7 +21,7 @@ class QualysWas < Kenna::Toolkit::BaseTask
 
       IGNORE_STATUS = {
         "false_positive" => "false positive",
-        "risk_accepted" => "risk accepted",
+        "risk_accepted" => "new", #TODO Changes the as per the status
```

Comment:
Getting below error 

> Invalid JSON caused Qualys WAS to fail. The JSON was improper because of: The property '#/assets/3/findings/5/triage_state' risk accepted is not valid finding-status

Reply:
https://help.kennasecurity.com/hc/en-us/articles/360026413111-Kenna-Data-Importer-JSON-Connector- false_positive and risk_accepted need underscores to be valid

Reply:
@kdavesecurview

Reply:
Done